docs/docs/quickstart/javascript.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: JavaScript
+hide_title: true
+---
+
+<h1 style={{ display: 'flex', gap: "2rem", alignItems: 'center' }}>Quickstart: JavaScript <img alt="npm" src="https://img.shields.io/npm/v/@simpleweb/open-format"/></h1>
+
+The Open Format JavaScript SDK provides a simple way to interact with Open Format.
+
+Don't forget to checkout our [React SDK](./react) which simplifies using the SDK even further.
+
+## Installation
+
+You can install the SDK with NPM or Yarn.
+
+```bash npm2yarn
+npm install @simpleweb/open-format ethers
+```
+
+## Initialising the SDK
+
+You can initialise the SDK by creating a new instance of `OpenFormatSDK` and selecting one of the available [networks](/glossary#networks) that are supported. This will create a new instance in "read-only" mode, meaning you can read from the [subgraph](/glossary#subgraph) but any interactions that require you to sign a transaction (deploying a new instance of the contract, minting an NFT, etc...) will throw an error.
+
+```tsx
+import { OpenFormatSDK } from '@simpleweb/open-format';
+
+const sdk = new OpenFormatSDK({
+  network: 'mumbai',
+});
+```
+
+If you wish to do more than just interact with the [subgraph](/glossary#subgraph), you will have to provide a [`signer`](/glossary#signer) when initialising the SDK.
+
+The easiest way to do this is by passing your wallet's private key as the [`signer`](/glossary#signer). You'd typically do this only from a Node.js context as to not expose your private key to the world.
+
+```tsx
+import { OpenFormatSDK } from '@simpleweb/open-format';
+
+const sdk = new OpenFormatSDK({
+  network: 'mumbai',
+  signer: '0x...',
+});
+```
+
+:::danger private keys
+
+**NEVER** pass a private key when initialising the SDK in the client, as this is something that you **DO NOT** want to expose to the world.
```

Comment:
Might be worth looking at re-wording this in the future (I appreciate that this is my copy)

Reply:
Let's leave this for now. I will add to backlog.